src/app/components/header/header.component.html

```diff
@@ -49,15 +35,18 @@
 <div class="border-bottom pb-lg-4 pb-3">
   <nav class="navbar navbar-expand-lg navbar-light navbar-default pt-0 pb-0">
     <div class="container px-0 px-md-3">
-      <div class="offcanvas offcanvas-start p-4 p-lg-0">
+      <!-- mobile menu tu-->
+      <div class="offcanvas offcanvas-start p-4 p-lg-0" [ngClass]="{show:!collapsed}">
         <div class="d-flex justify-content-between align-items-center mb-2 d-block d-lg-none"> <a routerLink="/"><img
               src="/assets/images/logo/freshcart-logo.svg" alt="eCommerce HTML Template"></a> <button type="button"
-            class="btn-close" aria-label="Close"></button> </div>
-        <div class="d-block"> </div>
-        <ul class="nav">
-          <li *ngFor="let category of categoriesList$ | async" class="nav-item"><a
-              [routerLink]="'categories/'+ category.id" class="nav-link">{{category.name}}</a></li>
-        </ul>
+            class="btn-close" aria-label="Close" (click)="collapsed = !collapsed"></button> </div>
```

Comment:
this works, but i would encourage you to use subject property instead of template variable